enterprise-metrics/jsonnetfile.json

```diff
@@ -36,15 +36,6 @@
         }
       },
       "version": "master"
-    },
-    {
```

Comment:
For anyone interested why there is only a deletion, the new library is already vendored at https://github.com/grafana/jsonnet-libs/blob/d91bf86b29b7ce0529e08042154b6f1382b8198f/enterprise-metrics/jsonnetfile.json#L22-L30